app/components/TransactionModal.tsx

```diff
@@ -34,7 +37,14 @@ export const TransactionModal = ({
   return (
     <Dialog open={isModalOpen(ModalId.WAITING_TRANSACTION)} onOpenChange={close}>
       <DialogContent>
-        <DialogHeader />
+        <DialogHeader>
+          <VisuallyHidden asChild>
+            <>
```

Comment:
Do we need the Fragment?

Reply:
Yes :c

`VisuallyHidden` can only have 1 children, that's why I used a Fragment to wrap both the Title and the Description